4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-scripting-gdscript-abstract-classes-and-methods.md

---
type: entry
section: scripting
subsection: gdscript
rank: 1
importance: 3
anchor: abstract-classes-and-methods
title: Abstract classes and methods
blockquote: U Can’t Touch This
text: |

>   You can now declare GDScript classes to be abstract. Declaring a class abstract means that the class is not meant to be instantiated directly. That means that you can prevent instances of a class, let’s say, `Animal`, that doesn’t mean anything itself other than to be extended by "concrete" classes like `Cat` and `Dog`.

Reviewer: ```suggestion
  You can now declare GDScript classes to be abstract. Declaring a class abstract means that the class is not meant to be instantiated directly. That means that you can prevent instances of a class, let’s say, `Animal`, that doesn’t have any purpose on its own other than to be extended by "concrete" classes like `Cat` and `Dog`.
```
I'd say
